test-framework/common/src/main/java/io/quarkus/test/common/configuration/InMemoryConfigSource.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2019 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.quarkus.test.common.configuration;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.eclipse.microprofile.config.spi.ConfigSource;
+
+/**
+ * In Memory Config Source that is used to store dynamically values from test.
+ */
+public class InMemoryConfigSource implements ConfigSource {
+
+    static final String NAME = "MemoryConfigSource";
+    private static final Map<String, String> PROPERTIES = new HashMap<>();
```

Comment:
Why is this static, when none of the methods that access it are static?

Reply:
I cannot remember now, too long ago and I am getting old. But probably I need it for a test, then I changed the approach and I miss the `static`. Changed now.